tools/plugins/com.liferay.ide.portlet.core/src/com/liferay/ide/portlet/core/operation/NewPortletClassDataModelProvider.java

```diff
@@ -1024,7 +1024,7 @@ else if( CREATE_RESOURCE_BUNDLE_FILE_PATH.equals( propertyName ) )
             }
             else
             {
-                return PortletCore.createWarningStatus( Msgs.resourceBundleFilePathEndWithProperties );
```

Comment:
This is a warning on purpose in case we are wrong and the user wants to go ahead and generate on like this.